### DISCUSSION TITLE
[Bug]: Validation rule prohibited is not the same with documentation

### DISCUSSION THREAD

**Discussion by yob-yob** [laravel/framework#39044] [Q&A]
on this code the comment say's

https://github.com/laravel/framework/blob/b5a70e2f0155e5ea095b6d71fa8e2894c5593d5c/src/Illuminate/Validation/Concerns/ValidatesAttributes.php#L1482-L1493

> Validate that an attribute does not exist.

but on the [documentation for rule-prohibited](https://laravel.com/docs/8.x/validation#rule-prohibited) it say's

> The field under validation must be empty or not present.

as a developer I expected that the word `OR` in `must be empty or not present` means that I can still submit the form with the prohibited input but it must not have a value....

P.S. if this is not a `bug` but rather a documentation change, then please do change the title of this issue, Thank you...

**Comment by liamoreilly** [comment]
Hi,

I also ran into this today as well. I have some complex validation rules and want to prevent a user from filling in a field based on the value of some other field. The prohibited rule in the documentation clearly states empty is allowed. My input is empty, which changes to null with the TrimStrings and ConvertEmptyStringsToNull middleware. Thus, I was expecting Prohibit to pass when the filed is not present or is null.

This is at minimum a miss-match with the documentation.

Thanks.
